ros/cpp/src/ptg_datahub/CMakeLists.txt

```diff
@@ -15,18 +15,26 @@ find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(std_msgs REQUIRED)
 find_package(sensor_msgs REQUIRED)
-
-add_executable(talker src/publisher_member_function.cpp)
-ament_target_dependencies(talker rclcpp std_msgs sensor_msgs)
-
-add_executable(listener src/subscriber_member_function.cpp)
-ament_target_dependencies(listener rclcpp std_msgs sensor_msgs)
-
-
+find_package(geometry_msgs REQUIRED)
+find_package(shape_msgs REQUIRED)
+find_package(angel_msgs REQUIRED)
+find_package(cv_bridge REQUIRED )
+find_package(OpenCV REQUIRED COMPONENTS core )
+
+add_executable(talker src/ptg_datahub.cpp)
+ament_target_dependencies(talker 
```

Comment:
This node could likely be renamed to "datahub" as well instead of the "hello world" "talker" example name. I mostly just don't think "talker" is appropriate anymore.